lib/ASTGen/Sources/ASTGen/Misc.swift

```diff
@@ -75,3 +61,39 @@ extension String {
     }
   }
 }
+
+extension ASTGenVisitor {
```

Comment:
Sounds good. Do you mind keeping `Misc.swift` for misc visits?

Reply:
If you strongly prefer, personally I think `ASTGen.swift` is perfectly reasonable for them. Ideally we wouldn't have a `Misc.swift` since it ends up being a dumping ground 😅